process/graduation_criteria.md

```diff
@@ -0,0 +1,118 @@
+# Azure Container Upstream Graduation Criteria v0.1
+The Azure Container Upstream Graduation Criteria owes much of its basis on two existing documents,
+[CNCF Graduation Criteria](https://github.com/cncf/toc/blob/fc161c13a86ba022277a906a10ff51a568406f7c/process/graduation_criteria.adoc) 
+and [.NET Foundation Project Maturity Profile](https://github.com/dotnet-foundation/project-maturity-model/blob/65fdfa43d7e68845bd65638c378cf834d7158b90/maturity-profiles.md). The goal of this document is to describe the process and 
+minimum bar for each increasing level of project maturity to both provide a
+goal for the project maintainers, but also to set expectations for consumers of these
+projects.
+
+## Process
+
+Each project in Container Upstream (AzCU) has an associated maturity level. A proposed AzCU project 
+should enter at the project's preferred maturity level. If the project originates within the AzCU 
+group, then it should begin at the Sandbox Stage. If a joining project originates outside of the 
+AzCU group, the preferred maturity level of the project must meet a 2/3 supermajority vote by the 
+AzCU technical oversight committee (TOC). If the vote tally can not be met at the preferred 
+maturity level, each previous level will receive a vote until a 2/3 supermajority is met. If the 
+project does not receive enough votes at the Sandbox stage, the project is rejected.
+
+Each project in AzCU will participate in a quarterly review by the TOC. The review will assess the 
+project's maturity level as well as its community momentum. A 2/3 supermajority vote is required to 
+graduate to the next maturity level. If the project is no longer needed or has become obsolete, the 
+TOC can also vote with a 2/3 supermajority to archive the project.
+
+For a project that has reached the "Graduation Stage" and would benefit from a more neutral home in 
+a community foundation, the option of project donation is an option.
+
+![AzCU Maturity Process](../images/maturity-diagram.png)
+
+## Sandbox Stage
+Sandbox projects are "early stage" projects that warrant experimentation. The project should be 
+beneficial to Azure and the Cloud Native community, open to contributions from the public and have 
+well-founded aspiration.
+
+The AzCU Sandbox is the entry point for early stage projects and has the following goals:
+- Encourage public visibility of experiments or other early work that can add value to the mission 
+  of AzCU, Cloud Native in Azure and its impact on the larger Cloud Native community.
+- Facilitate alignment with existing projects within Azure as well as the larger community
+- Remove obstacles to adoption and contribution by ensuring projects adhere to Microsoft Open Source
+  Guidelines, code of conduct and IP policy requirements
+  
+### Entry Requirements
```

Comment:
Do we expect to produce releases for things in the sandbox stage or expect people to build from source?
The incubation section only mentions 'Milestones for releases' so I was not sure where we should intent to introduce a release process.
To me milestones for releases is a step after having a release proceses :)

Reply:
> expect to produce releases for things in the sandbox stage 

IMO this should be part of the sandbox stage.